### PR TITLE
Displaying errors

### PR DESCRIPTION
**Pull request by nieznanysprawiciel**
First part of displaying errors task - CGI-27.
Frontend dipslays results of input video metadata validation

### CODE REVIEW

File: TranscodingDemo/Server/static/js/index.js [hunk 1]
```diff
@@ -119,14 +125,45 @@ function showFileInfo(videoMetadata){
...
+
+function setWarning(message){
+    $("#warning-placeholder").replaceWith(showUploadCompletedError(status, reponseText));
+    attachUploadLink();
+}
+
+function clearWarning(){
+    let warningPlaceholder = `
+        <div id="warning-placeholder" ></div>
+    `;
```
Comment (weaselix) [inline TranscodingDemo/Server/static/js/index.js]: Why do you use back quotes here? Backquotes have special meaning (literal templates). You do not need them here.

File: TranscodingDemo/Server/rpc.py [hunk 2]
```diff
@@ -8,6 +8,11 @@
 
 logger = logging.getLogger(__name__)
 
+# Suppress warnings from asyncio module
+logging.getLogger("asyncio").setLevel(logging.CRITICAL)
+logging.getLogger("autobahn.asyncio.component.Component").setLevel(logging.CRITICAL)
```
Comment (weaselix) [inline TranscodingDemo/Server/rpc.py]: For my eye it is better to make it in `settings.py`

File: TranscodingDemo/Server/static/js/index.js (hunk 1 above)
Comment (nieznanysprawiciel) [reply TranscodingDemo/Server/static/js/index.js]: For compatibility with other templates that use it.

File: TranscodingDemo/Server/rpc.py (hunk 2 above)
Comment (nieznanysprawiciel) [reply TranscodingDemo/Server/rpc.py]: It's good idea. But doesn it mean, I must import asyncio before these lines in settings ?

Comment (nieznanysprawiciel) [reply TranscodingDemo/Server/rpc.py]: Done

Comment (weaselix) [reply TranscodingDemo/Server/rpc.py]: ok, it was changed

**Pull request merged**